src/adapters/abstract/TimelockedAdapter.sol

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright (c) 2025 Morpho Association
+pragma solidity >=0.8.28;
+
+import {IVaultV2} from "../../interfaces/IVaultV2.sol";
+import {ITimelockedAdapter} from "../../interfaces/ITimelockedAdapter.sol";
+
+abstract contract TimelockedAdapter is ITimelockedAdapter {
```

Comment:
fine, especially since it's in the adapters folder? I'm afraid one might think it modifies the behavior of the adapter or something
```suggestion
abstract contract Timelocks is ITimelocks {
```